support-frontend/assets/pages/digital-subscriber-thank-you/digitalSubscriptionThankYou.tsx

```diff
@@ -45,13 +44,11 @@ export function DigitalSubscriptionThankYou(): JSX.Element {
 	const paymentMethod = useContributionsSelector(
 		(state) => state.page.checkoutForm.payment.paymentMethod.name,
 	);
-	const { selectedAmounts, otherAmounts, billingPeriod } =
-		useContributionsSelector((state) => state.page.checkoutForm.product);
+	const { selectedAmounts, otherAmounts } = useContributionsSelector(
```

Comment:
It looks like there's quite a bit of Contributions/Supporter+ related logic copied over from the Support Checkout's Thank You page when this page was set-up for the `/kindle` route. We could probably review this and see whether any of this can be removed - not in this PR however, lets create a card to tidy this up.

Reply:
Specifically I'm thinking do we need this code to check `isOneOff` and `amountIsAboveThreshold`, that are passed to `getThankYouModuleData`.

I'm assuming `isOneOff` is always `false`? And `amountIsAboveThreshold` is always `true` when coming through the Digi Subs checkout? Can you confirm? If so, we could remove this code that calculates the values of these variables and just pass the boolean value to `getThankYouModuleData`.

Reply:
Yes can confirm, digitalEdition never one-off and cannot (currently) enter other amount to get below minimum in digital checkout (ie `isOneOff`=`false`, `amountIsAboveThreshold`=`true`). Have made this change suggested.